typescript/infra/scripts/helloworld/kathy.ts

```diff
@@ -110,20 +116,30 @@ async function main() {
       origin,
       remote: destination,
     };
-    const startTime = Date.now();
-    try {
-      await sendMessage(app, origin, destination, gasCalc);
-      log('Message sent successfully', { origin, destination });
-      messagesSendCount.labels({ ...labels, status: 'success' }).inc();
-    } catch (e) {
-      error(`Error sending message, continuing...`, {
-        error: format(e),
-        origin,
-        destination,
-      });
-      messagesSendCount.labels({ ...labels, status: 'failure' }).inc();
+
+    for (
+      let attempt = 1, needToRetry = true;
+      attempt <= MAX_SEND_RETRIES + 1 && needToRetry;
+      ++attempt
+    ) {
+      const startTime = Date.now();
+      try {
+        await sendMessage(app, origin, destination, gasCalc);
+        needToRetry = false;
+        log('Message sent successfully', { origin, destination, attempt });
+        messagesSendCount.labels({ ...labels, status: 'success' }).inc();
```

Comment:
can we just break out of the loop instead of this `needToRetry`

Reply:
If we did that we would not report the time taken.

Reply:
If we did that we would not report the time taken.